src/content/tutorials/observability/alerting/create-rules/_index.md

```diff
@@ -0,0 +1,112 @@
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    # This lets Alloy know to which tenant this alert belongs to
+    observability.giantswarm.io/tenant: my-team
+  name: cluster-resource-usage
+  namespace: my-namespace
+spec:
+  groups:
+  - name: cluster-resource-usage
+    rules:
+      - expr: |
+          avg by (cluster_id) (
+            node:node_cpu_utilization:ratio_rate5m
+          )
+        record: cluster:node_cpu:ratio_rate5m
+```
+
+## How to define an alerting rule
+
+[Alerting rules](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) allow you to define alert conditions based on Prometheus (or Loki) expression language expressions and to send notifications about firing alerts to an external service.
+
+Here is how you would define your alerting rule:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    # This lets Alloy know to which tenant this alert belongs to
+    observability.giantswarm.io/tenant: my-team
+  name: cluster-resource-usage-too-high
+  namespace: my-namespace
+spec:
+  groups:
+  - name: reliability
+    rules:
+      - alert: ComponentDown
+        annotations:
+          # See https://grafana.com/docs/grafana/latest/alerting/alerting-rules/templates/#template-annotations for useful annotations used by Grafana.
+          __dashboardUid__: my-dashboard-uid
+          summary: 'Component is down.'
+          description: 'Component {{ $labels.service }} is down.'
+          runbook_url: https://my-pretty-runbook
+        expr: count(up{job=~"component/.*"} == 0) by (cluster_id) > 0
+        # Alert if the expression returns results for more than 5 minutes
+        for: 5m
+        labels:
+          # This is a paging alert that need to wake my team up
+          severity: page
+```
+
+## Differentiate between metric vs Log-based rules
+
+Metric-based rules are evaluated by the Mimir ruler and are written using PromQL while Log-based rules are evaluated by the Loki ruler and are written using LogQL.
+
+To be able to differentiate if your alert is a log-based alert, you should label you alert with those labels:
+
+```yaml
+# This label is marked as deprecated but still needs to be configured.
+application.giantswarm.io/prometheus-rule-kind: loki
+observability.giantswarm.io/rule-type: logs
+```
+
+## Cluster vs installation scoping of rules
+
+To avoid conflicts when loading rules from multiple clusters in the same tenant (for instance, if your teams define a PrometheusRule in their application template deployed in a dev, staging and prod clusters), the Observability Platform support some level of scoping.
+
+Any rule deployed in workload clusters is by default cluster-scoped (only targets metrics coming from this cluster) while a rule deployed in management clusters is scoped to the installation (targets all metrics of all clusters).
```

Comment:
```suggestion
The Observability Platform supports some level of scoping to avoid conflicts when loading rules from multiple clusters in the same tenant (for instance, if your teams define a PrometheusRule in their application template deployed in dev, staging, and prod clusters).

Any rule deployed in workload clusters is, by default, cluster-scoped (it only targets metrics coming from this cluster), while a rule deployed in management clusters is scoped to the installation (it targets all metrics of all clusters).
```